Kitodo/src/test/java/org/kitodo/selenium/testframework/helper/WebDriverProvider.java

```diff
@@ -27,6 +31,12 @@
 public class WebDriverProvider {
 
     private static final Logger logger = LogManager.getLogger(WebDriverProvider.class);
+
+    // https://sites.google.com/a/chromium.org/chromedriver/downloads/version-selection
+    // Please don't rely on the LATEST_RELEASE file without a version suffix.
+    // It exists for backward compatibility only, and will be removed in the near future.
+    public static final String CHROME_DRIVER_LATEST_RELEASE_URL = "https://chromedriver.storage.googleapis.com/LATEST_RELEASE_75";
```

Comment:
For current usage is it not needed to made this class constant public. Maybe you can move this URI to this one used place and did not define it as a class constant.